assets/users/components/EditUser.tsx

```diff
@@ -25,16 +25,11 @@ import {getUserStateLabelDetails} from 'company-admin/components/CompanyUserList
 
 import {companyProductSeatsSelector, companySectionListSelector, sectionListSelector} from 'company-admin/selectors';
 import {IUser} from 'interfaces/user';
+import {IUserProfileStore, IUserProfileStore} from 'user-profile/reducers';
 
 const getCompanyOptions = (companies: any) => companies.map((company: any) => ({value: company._id, text: company.name}));
 
-interface IReduxStateProps {
-    allSections?: Array<any>;
-    companySections?: any;
-    seats?: any;
-}
-
-interface IProps extends IReduxStateProps {
+interface IProps extends IUserProfileStore {
```

Comment:
I'm not sure about this part, the component won't have access to the whole state, this should be extending the part returned from the mapStateToProps